src/dev/arm/smmu_v3.hh

```diff
@@ -90,7 +90,7 @@ class SMMUv3 : public ClockedObject
     const System &system;
     const MasterID masterId;
 
-    SMMUMasterPort    masterPort;
+    SMMUMasterPort    requestPort;
```

Comment:
Do you think that "mainPort" would be a better name? Maybe we need to get input from Arm folks to know for sure.

The other ports have more descriptive names... I wonder why this one's name wasn't descriptive...

Reply:
I don't really know what SMMUMasterPort does, so I don't think I know which one would be better. I think I changed that variable by accident because I thought it's related to MasterPort... Should I leave it as masterPort for now?

Reply:
I'm not sure, tbh. Changing the variable names is definitely harder (and more controversial). That said, we need to do it at some point and get feedback on our names. However, I just have no idea how this is used, so maybe we can skip it for now :)